templates/elasticsearch.yml.j2

```diff
@@ -33,6 +33,39 @@ action.auto_create_index: {{ es_action_auto_create_index }}
 
 {% if es_enable_xpack and es_api_basic_auth_username is defined and es_api_basic_auth_password is defined %}
 xpack.security.enabled: true
+
+{% if es_enable_transport_ssl | bool %}
```

Comment:
I think its ok to leave these autogenerated values in as long as there is a way to opt out of it by adding some kind of `es_enable_auto_ssl_configuration: false`. Right now if someone wanted to add multiple `certificate_authorities` (which is an array/list) they wouldn't be able to override it in `es_config` because those values are being provided first and assume that you only have one CA. There are probably a bunch of other edge cases that could prevent a user from enabling security if they were forced to have these exact values set.

Reply:
Good point, will add a `es_enable_auto_ssl_configuration` parameter and test.